OpenSim/Common/Storage.h

```diff
@@ -152,19 +152,19 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Storage, StorageInterface);
     // GET AND SET
     //--------------------------------------------------------------------------
     // SIZE
-    virtual int getSize() const { return(_storage.getSize()); }
+    virtual int getSize() const override { return(_storage.getSize()); }
```

Comment:
Harmless, but this shouldn't have both `virtual` and `override` (several more below).